<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonUI</class>
 <widget class="QDialog" name="PersonUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberPhoneInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressInput">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="personName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="personName_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="personName_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="personName_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="personName_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateInput">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="personName_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Birth</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPersonRole">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Client</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Employee</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Manager</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="personName_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPersonButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Person</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
